DSCResources/MSFT_xCertificateImport/MSFT_xCertificateImport.schema.mof

```diff
@@ -0,0 +1,8 @@
+[ClassVersion("1.0"), FriendlyName("xCertificateImport")]
+class MSFT_xCertificateImport : OMI_BaseResource
+{
+[Key] string Thumbprint;
+[Key] string Path;
+[write,ValueMap{"LocalMachine"},Values{"LocalMachine"}] string Location;
+[write] string Store;
```

Comment:
@allansson Yes, I believe those , would be the correct keys.  

Unfortunately updating xPfxImport would be a breaking change.  @HemantMahawar   Your thoughts on fixing xPfxImport?  [xCertificate on PowerShell Gallery](https://www.powershellgallery.com/packages/xCertificate/1.1.0.0)